server.js

```diff
@@ -153,7 +156,7 @@ function Server (opts) {
       if (req.method === 'GET' && (req.url === '/stats' || req.url === '/stats.json')) {
         infoHashes.forEach(function (infoHash) {
           var peers = self.torrents[infoHash].peers
-          var keys = Object.keys(peers)
+          var keys = Object.keys(peers.cache)
```

Comment:
This is a LRU instance not an array, we have to access the array to extract the keys.